core/modules/cart/test/unit/store/connectActions.spec.ts

```diff
@@ -0,0 +1,144 @@
+import * as types from '@vue-storefront/core/modules/cart/store/mutation-types'
+import config from 'config';
+import { StorageManager } from '@vue-storefront/core/lib/storage-manager'
+import { CartService } from '@vue-storefront/core/data-resolver'
+import cartActions from '@vue-storefront/core/modules/cart/store/actions';
+
+jest.mock('@vue-storefront/core/store', () => ({
+  dispatch: jest.fn(),
+  state: {}
+}));
+jest.mock('js-sha3', () => ({ sha3_224: jest.fn() }));
+jest.mock('@vue-storefront/i18n', () => ({ t: jest.fn(str => str) }));
+jest.mock('config', () => ({}));
+jest.mock('@vue-storefront/core/lib/multistore', () => ({
+  currentStoreView: jest.fn(),
+  localizedRoute: jest.fn()
+}));
+jest.mock('@vue-storefront/core/lib/logger', () => ({
+  Logger: {
+    log: jest.fn(() => () => {}),
+    debug: jest.fn(() => () => {}),
+    warn: jest.fn(() => () => {}),
+    error: jest.fn(() => () => {}),
+    info: jest.fn(() => () => {})
+  }
+}));
+jest.mock('@vue-storefront/core/data-resolver', () => ({ CartService: {
+  getCartToken: jest.fn()
+}}));
+jest.mock('@vue-storefront/core/lib/storage-manager', () => ({
+  StorageManager: {
+    get: jest.fn()
+  }
+}));
+jest.mock('@vue-storefront/core/app', () => ({ router: jest.fn() }));
+jest.mock('@vue-storefront/core/lib/search/searchQuery', () => jest.fn());
+jest.mock('@vue-storefront/core/helpers', () => ({
+  get isServer () {
+    return true
+  },
+  onlineHelper: {
+    get isOnline () {
+      return true
+    }
+  },
+  processLocalizedURLAddress: (url) => url
+}));
+
+const createContextMock = (props = {}) => ({
```

Comment:
As you use this on like all test in here you could move this into [test/unit/utils/index.ts](https://github.com/DivanteLtd/vue-storefront/blob/develop/test/unit/utils/index.ts).

When we do move this we should maybe add like a function to mock the logger so you have less boiler plate to do in most tests.